Updates the homepage to feature the new Connection pane

edgararuiz opened the pull request.

edgararuiz commented:
Deploy preview ready!

Built with commit 83fa742bd83d6d3bbd778b462d02706fbda5fa35

https://deploy-preview-36--rstudio-db.netlify.com

edgararuiz commented:
Click on the preview link to see the new homepage of db.rstudio.com: https://deploy-preview-36--rstudio-db.netlify.com

The changes have not been proofread yet.  I wanted to share with you this preview to make sure the message is on point.

Thank you

jmcphers commented on content/_index.Rmd:
```diff
@@ -1,50 +1,63 @@
+
+- **Actively developed, best-in-class packages**. These packages include: `dplyr`, `odbc`, `DBI`, `keyring`, `pool` and others.
+
+- **Integration with RStudio.**  Starting with version 1.1, it is now possible to perform tasks like navigate database objects inside the RStudio interface.
```
How about:

*RStudio 1.1 helps you write the R code you need to connect to a database, remembers connections you've made, and allows you to explore the objects and data in your database.*

jmcphers commented on content/_index.Rmd:
```diff
@@ -1,50 +1,63 @@
+  <img src="/ide/tags.png"   align="left">
 </div>
 
+The RStudio **Connections Pane** makes it possible to easily connect to a variety of data sources, and explore the objects and data inside the connection. It extends, and is designed to work with, a variety of other tools for working with databases in R. Also, preview the first 1,000 records in a database by simply clicking on the table icon.
```
I think just the first sentence is fine here since the rest is in the picture and/or already implicit in this doc.

edgararuiz commented:
- Made the changes @jmcphers suggested
- Added a new Connections page https://deploy-preview-36--rstudio-db.netlify.com/connections/
- Changed the link in the home page from the support page to the new internal Connection page 
- At @nwstephens request, made sure to add the items from the Troubleshooting article: https://deploy-preview-36--rstudio-db.netlify.com/connections/#new-connection-interface 
- Added a more robust illustration for the Connection pane: https://deploy-preview-36--rstudio-db.netlify.com/connections/#connections-pane

@hadley - @nwstephens asks if you can confirm that the message at the top of the home page (https://deploy-preview-36--rstudio-db.netlify.com/), more specifically, the 3 bullets related to how support Databases and R, is correct

Thanks!

hadley commented:
I'd title the bullets as:

* Developing best-in-class packages
* Integrating with RStudio
* Promoting best practices

I'd also remove "DBI" from the list of packages since RStudio isn't directly involved.
